<launch>

    <node pkg="abm_interaction" type="abm_cordial_client.py" name="abm_cordial_client" output="screen"/>

</launch>
